lib/git-subrepo

```diff
@@ -919,12 +934,17 @@ subrepo:commit() {
   local edit_flag=
   $edit_wanted && edit_flag=--edit
 
+  local no_verify_flag=' --no-verify'
+  $verify_wanted && no_verify_flag=''
+
   o "Commit to the '$original_head_branch' branch."
   if [[ $original_head_commit != none ]]; then
     if [[ $commit_msg_file ]]; then
-      RUN git commit $edit_flag --file "$commit_msg_file"
+      # shellcheck disable=2086
+      RUN git commit$no_verify_flag $edit_flag --file "$commit_msg_file"
     else
-      RUN git commit $edit_flag -m "$commit_message"
+      # shellcheck disable=2086
+      RUN git commit$no_verify_flag $edit_flag -m "$commit_message"
```

Comment:
To prevent disabling 2086 this should use Parameter Expansion https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#Shell-Parameter-Expansion-1